pkgs/by-name/cp/cppcodec/package.nix

```diff
@@ -16,6 +17,17 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-k4EACtDOSkTXezTeFtVdM1EVJjvGga/IQSrvDzhyaXw=";
   };
 
+  patches = [
+    # CMake 2.8 is deprecated and is no longer supported by CMake > 4
+    # https://github.com/NixOS/nixpkgs/issues/445447
+    # Luckily, this has been fixed on master.
+    (fetchpatch {
+      name = "modernize-cmake.patch";
+      url = "https://github.com/tplgy/cppcodec/commit/8019b8b580f8573c33c50372baec7039dfe5a8ce.patch";
+      hash = "sha256-0MCx3nTsey4Qonx+lyexbcxut0qIHOJZbkJ9u23Zuv8=";
+    })
+  ];
+
```

Comment:
It's perhaps a bit annoying that this patch also updates the version of the project.  But given that the change is from 0.1 to 0.3 and we're on 0.2, it won't be a new breakage.